tests/unit/source_code/samples/functional/file-access/complex-sql-notebook.sql

```diff
@@ -0,0 +1,60 @@
+-- Databricks notebook source
+-- MAGIC %md
+-- MAGIC #Test notebook for DBFS discovery in Notebooks
+
+-- COMMAND ----------
+-- DBTITLE 1,A Python cell that references DBFS
+-- MAGIC %python
+-- ucx[dbfs-usage:+1:7:+1:18] Deprecated file system path: dbfs:/...
```

Comment:
can you do a PR to PR to prove this works?

Reply:
I think it's already proven as part of this PR since `functional.py` succeeds in validating this sample. Do you have another type of proof in mind ?

Reply:
See #2528